examples/common_vmseries/variables.tf

```diff
@@ -605,6 +605,67 @@ variable "bootstrap_storages" {
   }))
 }
 
+variable "vmseries_common" {
+  description = <<-EOF
+  A map defining common settings for all created VM-Series instances. 
+  
+  It duplicates popular properties from `vmseries` variable, specifically `vmseries.image` and `vmseries.virtual_machine` maps.
+  However, if values are set in those maps, they still take precedence over the ones set within this variable. As a result, all
+  common properties can be overriden on a per-VM basis.
+
+  Following properties are supported:
+  
+  - `version`           - (`string`, optional) describes the PAN-OS image version from Azure Marketplace.
+  - `custom_id`         - (`string`, optional) absolute ID of your own custom PAN-OS image.
+  - `size`              - (`string`, optional, defaults to module default) Azure VM size (type). Consult the *VM-Series
+                          Deployment Guide* as only a few selected sizes are supported.
+  - `bootstrap_options` - (`string`, optional, mutually exclusive with `bootstrap_package`) bootstrap options passed to PAN-OS
+                          when launched for the 1st time, for details see module documentation.
+  - `bootstrap_package` - (`map`, optional, mutually exclusive with `bootstrap_options`) a map defining content of the bootstrap
+                          package. For details and available properties refer to `vmseries` variable.
+  EOF
+  default     = {}
+  type = object({
+    version           = optional(string)
+    custom_id         = optional(string)
```

Comment:
shall we rename it to sth like `custom_image_id` to make it clear what it refers to? Btw does it actually refer to the urn or the really long azure id which contains subscription etc with slashes?

Reply:
Ok, I'm just getting rid of `custom_id` completely as agreed.